package.json

```diff
@@ -134,6 +133,7 @@
     "nuclearnet.js": "^1.5.3",
     "pixi.js": "^4.8.7",
     "protobufjs": "^6.8.8",
+    "raw-loader": "^3.0.0",
```

Comment:
I believe this should remain in `devDependencies`, i.e. it is only used during building/development.